back/engines/commercial/insights/app/services/insights/xlsx_service.rb

```diff
@@ -0,0 +1,30 @@
+module Insights
+	class XlsxService
+
+		def generate_inputs_xlsx(inputs, categories, view_private_attributes = false)
```

Comment:
```suggestion
		def generate_inputs_xlsx(inputs, categories, view_private_attributes: false)
```

Usually, we use keyword arguments for boolean arguments. It is also more consistent with `XlsxService`.